util/regexp-assemble/data/932210.data

```diff
@@ -51,7 +57,8 @@ scanstats
 schema
 selftest
 separator
-sha3sum
```

Comment:
I think this is an accident (probably because of the regex you're using).

Reply:
You are right. Corrected regex.